UIKitHomework/UIKitHomework/MainViewController.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  MainViewController.swift
+//  UIKitHomework
+//
+//  Created by MacBook Эдуард on 05.07.2021.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
+
+// MARK: - Visual Elements
+    
+    private let logo = UIImageView()
+    private let minusImage = UIImageView()
```

Comment:
```suggestion
    private let minusImageView = UIImageView()
```
это не UIImage, а UIImageView